### PR TITLE
[issue-14] create Account Menu and Page

### PR DESCRIPTION
**Pull request by alecherryy**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
+import { AuthProvider } from './Auth';
```
Comment (rileyhgrant) [inline src/index.js:4]: ah cool, makes more sense to put it here!

File: src/stories/components/AccountMenu/AccountMenu.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+*/
+
+export const AccountMenu = () => {
+  const { logout, currentUser } = useContext(AuthContext);
+  const [profile, setProfile] = useState(null);
+  const history = useHistory();
+
+  useEffect(() => {
+    if (currentUser) {
+      // setError('');
```
Comment (rileyhgrant) [inline src/stories/components/AccountMenu/AccountMenu.js:26]: ah cool, so we'll add a setError at some point to show the user whatever error firebase auth spits back. word.

File: src/stories/components/AccountMenu/styles.scss [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+    &::before {
+      opacity: 0;
+    }
+
+    &::after {
+      opacity: 1;
+    }
+
+    .account-menu__text {
```
Comment (rileyhgrant) [inline src/stories/components/AccountMenu/styles.scss:76]: looks pretty sliiiick

**Review by rileyhgrant — APPROVED** [review]
Looks dope! Any comments I left are just comments haha. Very nicely done.

File: src/stories/components/MainMenu/MainMenu.js [hunk 4]
```diff
@@ -33,44 +32,31 @@ export const MainMenu = () => {
...
-  };
-
   return (
     <ul className="main-menu">
+      { currentUser ?
+        <li className="main-menu__item">
+          Welcome back,&nbsp;
+          <strong>{profile ?
+            profile.username :
+            currentUser.fullName
```
Comment (looper-m) [inline src/stories/components/MainMenu/MainMenu.js:42]: `currentUser` does not have a `fullName`. There is an attribute `displayName` but thats just null.
The profile always has a `username` and I left `email` in `else` just as a safety net

**Review by looper-m — CHANGES_REQUESTED** [review]

**Pull request merged**